core/src/main/java/jenkins/install/SetupWizard.java

```diff
@@ -95,6 +98,32 @@ public SetupWizard() {
 
     private static final Logger LOGGER = Logger.getLogger(SetupWizard.class.getName());
 
+    private static final String ADMIN_INITIAL_API_TOKEN_PROPERTY_NAME = SetupWizard.class.getName() + ".adminInitialApiToken";
+
+    /**
+     * This property determines the behavior during the SetupWizard install phase concerning the API Token creation 
+     * for the initial admin account.
+     * The behavior depends on the provided value:
+     * - [2-char hash version][32-hex-char of secret], where the hash version is currently only 11. 
+     *      E.g. 110123456789abcdef0123456789abcdef.
+     *      A fixed API Token will be created for the user with that plain value as the token.
+     *      It is strongly recommended to use it to generate a new one (random) and then revoke it.
+     *      See {@link ApiTokenProperty#generateNewToken(String)} and {@link ApiTokenProperty#revokeAllTokensExceptOne(String)}
+     *      for scripting methods or using the web API calls: 
+     *      /user/[user-login]/descriptorByName/jenkins.security.ApiTokenProperty/generateNewToken and 
+     *      /user/[user-login]/descriptorByName/jenkins.security.ApiTokenProperty/revokeAllExcept 
+     * - @[file-location] where the file contains plain text value of the token, all stuff explained above is applicable
+     *      The application will not delete the file after read, so the script is responsible to clean up the stuff
+     * - true
+     *      A token is generated using random value at startup and the information is put 
+     *      in the file "$JENKINS_HOME/secrets/initialAdminApiToken"
```

Comment:
There's a fourth state that needs to be described here, when the value is null. Otherwise, the description is overly confusing, particularly as that's the default state.

Reply:
> when the value is null

Not sure to follow what you mean by that. If the system property is not set, or set to null, there will be no action, as for most of such system property, the default behavior occurs.

> Otherwise, the description is overly confusing, particularly as that's the default state.

The default state is when there is no system property. 
Could you elaborate what is confusing in the description?

----

Adding some information to the JavaDoc.

Reply:
Thanks for the update!